concourse-irccat-resource.yml

```diff
@@ -0,0 +1,5 @@
+name: irccat-resource
+url: https://github.com/haiku/irccat-resource
```

Comment:
Do you mind adding `container_image` here. that is the docker image name or the url to pull the image from whatever repo.